Gopkg.toml

```diff
@@ -1,8 +1,8 @@
 # List of dependecies for CSI attacher
 
-[[constraint]]
+[[override]]
   name = "github.com/container-storage-interface/spec"
-  version = "1.0.0"
+  version = "=1.0.0"
```

Comment:
This should be updated to 1.1.0 to be consistent with csi-lib-utils 0.6.1.
"override" should be changed to "constraint".

Reply:
Upgrading to 1.1.0 breaks a bunch of tests which I don't want to be fixing in this PR, hence the override

Reply:
```
go test `go list ./... | grep -v 'vendor' `
# github.com/kubernetes-csi/external-attacher/vendor/github.com/kubernetes-csi/csi-test/driver
vendor/github.com/kubernetes-csi/csi-test/driver/mock.go:41:5: cannot use servers.Controller (type *MockControllerServer) as type csi.ControllerServer in field value:
	*MockControllerServer does not implement csi.ControllerServer (missing ControllerExpandVolume method)
vendor/github.com/kubernetes-csi/csi-test/driver/mock.go:42:5: cannot use servers.Node (type *MockNodeServer) as type csi.NodeServer in field value:
	*MockNodeServer does not implement csi.NodeServer (missing NodeExpandVolume method)
?   	github.com/kubernetes-csi/external-attacher/cmd/csi-attacher	[no test files]
FAIL	github.com/kubernetes-csi/external-attacher/pkg/attacher [build failed]
ok  	github.com/kubernetes-csi/external-attacher/pkg/controller	1.451s
make: *** [test-go] Error 2
```

I can fix this in a follow-up PR

Reply:
That should go away if you also update csi-test to 2.0.0.

Reply:
ah, thanks, updated now